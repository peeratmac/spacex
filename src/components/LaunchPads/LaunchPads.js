import React from 'react';

export const LaunchPads = () => {
  return (
    <div>
      <h1>Under Construction, please check back later! Thank you!</h1>
    </div>
  );
};

export default LaunchPads;
