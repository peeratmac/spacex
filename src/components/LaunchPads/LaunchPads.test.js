import React from 'react';
import { shallow } from 'enzyme';
import { LaunchPads } from './LaunchPads';

describe('LAUNCH PADS', () => {
  it('should match the LaunchPads Snapshot UI', () => {
    let wrapper = shallow(<LaunchPads />);

    expect(wrapper).toMatchSnapshot();
  });
});
