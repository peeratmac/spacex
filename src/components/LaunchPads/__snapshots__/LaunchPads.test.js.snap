// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LAUNCH PADS should match the LaunchPads Snapshot UI 1`] = `
<div>
  <h1>
    Under Construction, please check back later! Thank you!
  </h1>
</div>
`;
